app/src/main/java/org/mozilla/fenix/FeatureFlags.kt

```diff
@@ -87,7 +87,7 @@ object FeatureFlags {
     /**
      * Enables showing the wallpaper functionality.
      */
-    val showWallpapers = Config.channel.isNightlyOrDebug
```

Comment:
I think we can remove this flag entirely now, right? By the time this branch hits release, we'll want it available to that channel as well

Reply:
Yeah, we can I just wanted to be extra careful in case we found something that we don't like and it won't go automatically to release. But lets be optimistic 🤞🏽. I'll updated it :)